hw/lint/doc/README.md

```diff
@@ -1,18 +1,93 @@
+However, the _"Low Risc Lint Rules"_ are available as part of the default policies in AscentLint release 2019.A.p3 or newer (as `LRLR-v1.0.policy`).
+This enables designers that have access to this tool to run the lint flow provided locally on their premises.
+
+In order to run lint on a comportable IP block, the corresponding Fusesoc core file must have a lint target and include (optional) waiver files as shown in the following example taken from the Fusesoc core of the AES comportable IP:
+```
+filesets:
+
+  [...]
+
+  files_verilator_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+    files:
+      - lint/aes.vlt
+    file_type: vlt
+
+  files_ascentlint_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+    files:
+      - lint/aes.waiver
+    file_type: waiver
+
+  [...]
+
+targets:
+  default: &default_target
+    filesets:
+      - tool_verilator  ? (files_verilator_waiver)
+      - tool_ascentlint ? (files_ascentlint_waiver)
+      - files_rtl
+    toplevel: aes
+
+  lint:
+    <<: *default_target
+    default_tool: verilator
+    parameters:
+      - SYNTHESIS=true
+    tools:
+      verilator:
+        mode: lint-only
+        verilator_options:
+          - "-Wall"
+```
+Note that the setup shown above also supports Verilator lint as a fall-back option that is open source.
+The same lint target is reused however for both AscentLint and Verilator (we override the tool selection when invoking Fusesoc).
+Lint waivers can be added to the flow by placing them in the corresponding waiver file.
+In this example this would be `lint/aes.waiver` for AscentLint and `lint/aes.vlt` for Verilator.
+
+In order to manually run lint on a specific block, make sure AscentLint is properly installed and step into the `hw/lint` folder.
+The makefile in that folder contains all targets that can be manually invoked.
+For example, to run lint on AES, do:
+```
+make ip-aes_lint
+```
+This run will exit with PASSED status on the command line if there are no lint errors or warnings.
+Otherwise it will exit with ERROR status, in which case you can get more information by running
+```
+make clean
+make ip-aes_lint
+make report
+```
+In order to build all lint targets and produce a summary report, the `make all` target can be invoked.
+For more detailed information on a particular lint run you can inspect the tool output inside the build folder that is created by Fusesoc.
+
+Note that all AscentLint targets have a Verilator counterpart that is suffixed with `_vlint` instead of `_lint`.
+This enables designers without access to AscentLint to iterate with an open-source tool before making their first Pull Request.
+
+For batch regressions we have integrated this flow into the `atom` tool, which can be invoked as follows:
```

Comment:
Maybe we should state here that this is to be executed from within the top project directory?

Reply:
oh yes thanks. I've added that to the explanation.